### PR TITLE
Add vehicle stance and drift tires

### PR DESCRIPTION
**Pull request by AlexR32**
added new functions from los santos tuners DLC
drift tires in vehicle mod menu
vehicle stance in personal vehicle menu

### CODE REVIEW

File: vMenu/menus/PersonalVehicle.cs [hunk 1]
```diff
@@ -39,6 +39,7 @@ private void CreateMenu()
             MenuItem setVehice = new MenuItem("Set Vehicle", "Sets your current vehicle as your personal vehicle. If you already have a personal vehicle set then this will override your selection.") { Label = "Current Vehicle: None" };
             MenuItem toggleEngine = new MenuItem("Toggle Engine", "Toggles the engine on or off, even when you're not inside of the vehicle. This does not work if someone else is currently using your vehicle.");
             MenuListItem toggleLights = new MenuListItem("Set Vehicle Lights", new List<string>() { "Force On", "Force Off", "Reset" }, 0, "This will enable or disable your vehicle headlights, the engine of your vehicle needs to be running for this to work.");
+            MenuListItem toggleStance = new MenuListItem("Vehicle Stance", new List<string>() { "Default", "Lowered" }, 0, "This will increase or decrease your car suspension. Note this will work only with los santos tuners cars.");
```
Comment (cm8263) [inline vMenu/menus/PersonalVehicle.cs:42]: Since there are only two possible values, it might make more sense to change this to a `MenuCheckboxItem` and reword it to something like "Toggle Lowered Suspension".

**Review by cm8263 — COMMENTED** [review]
I'm curious why you've added the Suspension to the Personal Vehicle menu, and the Tires to the Vehicle Mod menu. Not suggesting you change them necessarily, just want to know the thought process behind it.

File: vMenu/menus/PersonalVehicle.cs (hunk 1 above)
Comment (AlexR32) [reply vMenu/menus/PersonalVehicle.cs:42]: > it might make more sense to change this to a MenuCheckboxItem

it was list in gta online menu and i just copied it from it
> I'm curious why you've added the Suspension to the Personal Vehicle menu, and the Tires to the Vehicle Mod menu.

because i think suspension more related to personal vehicle settings because it suppose to be changed with car keys
and drift tires was in car meet mechanic menu so i just added them to mod menu

Comment (cm8263) [reply vMenu/menus/PersonalVehicle.cs:42]: Ah, that makes sense; I haven't played GTA:O so that would be the context I was missing.

**Comment by TomGrobbe** [comment]
I believe all these features have just been merged in PR #290 so I'm closing this one. If I've missed anything feel free to rebase and re-open this PR.
